mitodl/ocw-hugo-themes
# sponsor logos updated

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes https://github.com/mitodl/ocw-hugo-themes/issues/586

#### What's this PR do?
- Remove TelMex logo
- Add The Andrew W. Mellon Foundation logo
- Add The William and Flora Hewlett Foundation logo 
- Change "Our Corporate Sponsors" to "Our Corporate and Foundation Supporters"
- logos in this order: Ab Initio -- MathWorks -- The William and Flora Hewlett Foundation -- The Andrew W. Mellon Foundation
- Sets  `width: 180px;` and `height: 45px;` for all the sponsor images.

#### How should this be manually tested?
- Go to www homepage (preview [netlify deployed version](https://ocw-hugo-themes-pr-595--ocw-next.netlify.app/))
- Verify the above points (or the points mentioned in issue ticket)
- Verify that logos are responsive for laptop, desktop, tablet, and mobile screens

#### Screenshots (if appropriate)
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/93309234/161696584-c345bcfc-a392-4500-a603-109c882f27ce.png">

<img width="841" alt="image" src="https://user-images.githubusercontent.com/93309234/161696760-c4162149-1e5a-428d-876e-fc220adfe7ed.png">

<img width="899" alt="image" src="https://user-images.githubusercontent.com/93309234/161696669-c7d7c374-e719-475a-a6a1-b59a1bb97cb0.png">

<img width="350" alt="image" src="https://user-images.githubusercontent.com/93309234/161696718-1aba4b96-a63f-47e3-9c67-caaec0a62e58.png">
